packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -273,6 +296,7 @@ class AnalyzeCommand extends FlutterCommand {
       new RegExp(r'\[lint\] Prefer using lowerCamelCase for constant names.'), // sometimes we have no choice (e.g. when matching other platforms)
       new RegExp(r'\[lint\] Avoid defining a one-member abstract class when a simple function will do.'), // too many false-positives; code review should catch real instances
       new RegExp(r'[0-9]+ (error|warning|hint|lint).+found\.'),
+      new RegExp(r'\[info\] TODO.+'),
```

Comment:
Flutter
Repository: <unknown>
Branch: 
Revision: 90f1e5b250966e43e24968d03829b85b7e27ac64 (43 seconds ago)

and for dart:

```
Dart VM version: 1.14.0-dev.5.0 (Fri Dec 18 02:14:17 2015) on "macos_x64"
```